draft-tigress-requirements.md

```diff
@@ -123,7 +123,7 @@ General terms:
 
 - [Req-AnyPlatorm] Solution SHOULD be able to communicate with any mobile devices of any operating system and allow easy implementation of server-side components without requiring a specific Cloud stack.
 - [Req-NontechnicalUX] Solution SHALL enable secure credential transfer for non technical users.
-- [Req-SmoothUX] Solution SHALL allow for user experience where neither Sender nor Receiver is presented with raw data required only by the secure transfer protocol. Data that is only parsed programmatically and not required to be presented to the end user should never be visible to said user in whichever messaging application the sender chose to initiate the transfer on. This eliminates the possibility of merely sending the requisite data inline, through an SMS or email for example, rather than leveraging an Intermediary server.
+- [Req-SmoothUX] Solution SHALL allow for user experience where neither Sender nor Receiver is presented with raw data required only by the secure transfer protocol. Data that is only parsed programmatically and not required to be presented to the end user and should never be visible to said user in whichever messaging application the sender chose to initiate the transfer on. This eliminates the possibility of merely sending the requisite data inline, through an SMS or email for example, rather than leveraging an Intermediary server.
```

Comment:
`Solution SHALL allow for user experience where neither Sender nor Receiver is presented with raw data required only by the secure transfer protocol.`, sounds a bit awkward, but wasn't too sure what point you wanted to make with the second half of the sentence.

Reply:
I am not an expert in punctuation, but logically, and is not required here, since it changes the logic of the sentence.
We could possibly use commas to separate a logic block like:
"Data, that is only parsed programmatically and not required to be presented to the end user, should never be visible to said user ..."

Reply:
I think the sentence needs to be either the original or split into two sentences to be clearest:

The data is only parsed programmatically and not required to be presented to the end user. This data should never be visible to said user in whichever messaging application the sender chose to initiate the transfer on.

Otherwise, it is a bit of a run on sentence.

Reply:
@dimmyvi I think we need to remove the sentence @nicksha mentioned in his comment as that is not true. On android, there are ways to literally take the data and manually add it (potentially from a third party app).

Reply:
We can keep the requirement, just wanted to clarify the sentence.  We also have limitation when it comes to third party apps, we won't be able to just extract provisioning information.

Reply:
Split the sentence into 2 per Casey's suggestion